vignettes/tutorial-create-your-first-rhino-app.Rmd

```diff
@@ -7,11 +7,28 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+<style>
+
+.app-preview {
+
```

Comment:
I'd suggest to use vertical whitespace sparingly - it doesn't really aid readability when used at the beginning/and of code blocks. See [here](https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace) for additional commentary - it's Google's C++ style guide, but the comments apply to other languages as well.

Reply:
Noted